packages/devtools/pack-extension.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const zip = require('bestzip');
+
+const DIST_PATH = path.resolve(__dirname, '../../dist/packages/devtools');
+
+if (!fs.existsSync(DIST_PATH)) {
+  console.error(`"${DIST_PATH}" does not exist, please run "build" command first`);
```

Comment:
is this command necessary if the `build` command is an nx dependency ?

Reply:
I think it doesn't hurt. It could be useful when the build output directory changes.